README.md

```diff
@@ -214,6 +218,37 @@ curl https://YOUR_DOMAIN.fakturownia.pl/invoices.json \
         }
     }'
 ```
+<b>Klient: nowy lub istniejący </b></br>
+Podczas dodawania nowego dokumentu system automatycznie stara się dopasować przesłane dane nabywcy (pola zaczynające się od słowa "buyer") z istniejącym na koncie klientem. Jeśli nie uda się dopasować żadnego istniejącego klienta, system utworzy nowego. Gdy dodajemy dokument dla klienta, którego mamy już w bazie, zamiast pól zaczynających się od słowa "buyer" zaleca się używanie samego identyfikatora klienta - "client_id".
```

Comment:
stara się dopasować po jakich atrybutach? wydaje mi się, że tylko po nazwie

Reply:
Przy wyszukiwaniu klienta udział biorą w zasadzie 3 atrybuty: name, tax_no, email (lub w przypadku osoby prywatnej mogą mieć znaczenie również first_name, last_name).

Reply:
To może coś takiego wystarczy _"Podczas dodawania nowego dokumentu system automatycznie stara się dopasować przesłane dane nabywcy z istniejącym na koncie klientem. W dopasowaniu biorą udział następujące pola: "buyer_name", "buyer_tax_no", "buyer_email" (i/lub dla klienta prywatnego "buyer_first_name" i "buyer_last_name") Jeśli nie uda się dopasować... "_ ?